lib/draper/test_case.rb

```diff
@@ -29,14 +29,10 @@ module Behavior
   end
 end
 
-if defined?(ActionController::TestCase)
-  class ActionController::TestCase
-    include Draper::TestCase::ViewContextTeardown
-  end
+class ActionController::TestCase
+  include Draper::TestCase::ViewContextTeardown
 end
 
-if defined?(ActionMailer::TestCase)
```

Comment:
My mistake, I didn't realize that this file was something that other people's test suites relied upon. I'll undo this change.